tests/e2e/k8s/util/kind.go

```diff
@@ -361,6 +369,14 @@ func (c *KindCluster) CreateFromYAML(yaml, namespace string) error {
 		decoder.MutateNamespace(namespace))
 }
 
+// CreateFromFolder creates k8s objects from a yaml in a folder.
+func (c *KindCluster) CreateFromFolder(folder string) error {
+	pattern := "*"
+	fs := os.DirFS(folder)
```

Comment:
I would remove this lines, and embed the values directly to the function call below.